packages/extension/src/index.ts

```diff
@@ -19,12 +19,14 @@ export function activate (context: vscode.ExtensionContext) {
      */
     marquee: {
       setup: (tangle: Client<{ counter: number }>) => {
-        let i = 0
-        setInterval(() => {
-          tangle.emit('counter', ++i)
-        }, 1000)
-      }
-    }
+        tangle.whenReady().then(() => {
```

Comment:
We will have to update our [internal mock](https://github.com/stateful/vscode-marquee/blob/main/packages/extension/tests/__mocks__/tangle/webviews.ts) so that the unit test passes.

Reply:
Will do.